cvat-ui/src/components/dashboard/content/dashboard-content.tsx

```diff
@@ -79,15 +81,13 @@ class DashboardContent extends Component<any, any> {
   }
 
   private onDeleteTask(task: any) {
```

Comment:
Why don't setup specific type (window.cvat.classes.Task)?

Reply:
It should be declared on `cvat-core` side and then can be imported and used here. I can declare types on client side but this approach will work until something changed on `cvat-core`.